x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/settings_flyout/schedule/create_flyout/index.tsx

```diff
@@ -50,7 +50,7 @@ export const CreateFlyout: React.FC<Props> = React.memo(({ onClose }) => {
     http,
   });
 
-  const { sourcererDataView } = useSourcererDataView();
+  const { indexPattern } = useSourcererDataView();
```

Comment:
Similarly to https://github.com/elastic/kibana/pull/205137#discussion_r1896745964

In `main`, the `convertToBuildEsQuery` function called here accepts a parameter named `dataViewSpec`, which is typed as:

```typescript
dataViewSpec: DataViewSpec | undefined;
```

In `8.x`, the `convertToBuildEsQuery` function takes a different parameter named `indexPattern` instead of `dataViewSpec`, and it's typed differently:

```typescript
indexPattern: DataViewBase | undefined;
```

The page was updated to call the `convertToBuildEsQuery` function using the `8.x` version of the parameter.